packages/fxa-settings/src/components/NotificationPromoBanner/index.stories.tsx

```diff
@@ -48,10 +48,3 @@ export const NotVisible = () => (
     {...disabledNotificationProps}
   ></NotificationPromoBanner>
 );
-
-export const Stacked = () => (
```

Comment:
Spacing between sections/banners/etc. now controlled at the PageSettings level, no need for this story.